2/index.js

```diff
@@ -0,0 +1,14 @@
+console.log("No.2");
+// Main-Program
+// node作成
+const list = document.querySelector("main > ul");
```

Comment:
IMO.
I think it would be better to specify the id directly to the `ul `tag.
This is because it can accommodate changes in HTML structure.